.github/pull_request_template.md

```diff
@@ -1,18 +1,19 @@
-Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.
+
```

Comment:
new line?

Reply:
I had copy/pasted the exact same template from admiral, I realize now they were actually different?